Ryujinx.HLE/HOS/Services/Fs/FileSystemProxy/FileSystemProxyHelper.cs

```diff
@@ -111,13 +117,13 @@ public static ResultCode OpenFileSystemFromInternalFile(ServiceCtx context, stri
                     PartitionFileSystem nsp = new PartitionFileSystem(pfsFile.AsStorage());
 
                     ImportTitleKeysFromNsp(nsp, context.Device.System.KeySet);
-                    
+
                     string filename = fullPath.Replace(archivePath.FullName, string.Empty).TrimStart('\\');
 
-                    if (nsp.FileExists(filename))
-                    {
-                        return OpenNcaFs(context, fullPath, nsp.OpenFile(filename, OpenMode.Read).AsStorage(), out openedFileSystem);
-                    }
+                    Result rc = nsp.OpenFile(out LibHac.Fs.IFile ncaFile, filename, OpenMode.Read);
```

Comment:
```suggestion
                    Result resultCode = nsp.OpenFile(out LibHac.Fs.IFile ncaFile, filename, OpenMode.Read);
```

and add an empty line after.